man/git-authors.1

```diff
@@ -1,87 +1,38 @@
-.\" generated with Ronn/v0.7.3
-.\" http://github.com/rtomayko/ronn/tree/0.7.3
-.
-.TH "GIT\-AUTHORS" "1" "October 2017" "" "Git Extras"
-.
+.\" generated with Ronn-NG/v0.9.1
+.\" http://github.com/apjanke/ronn-ng/tree/0.9.1
+.TH "GIT\-AUTHORS" "1" "November 2023" "" "Git Extras"
 .SH "NAME"
 \fBgit\-authors\fR \- Generate authors report
-.
 .SH "SYNOPSIS"
 \fBgit\-authors\fR [\-l, \-\-list] [\-\-no\-email]
-.
 .SH "DESCRIPTION"
-Populates the file matching \fIauthors|contributors \-i\fR with the authors of commits, according to the number of commits per author\. Opens the file in \fB$EDITOR\fR when set\.
-.
+.TS
+allbox;
+Populates the file matching _authors	contributors \-i_ with the authors of commits, according to the number of commits per author\.
+.TE
 .P
 See the "MAPPING AUTHORS" section of \fBgit\-shortlog\fR(1) to coalesce together commits by the same person\.
-.
 .SH "OPTIONS"
 \-l, \-\-list
-.
 .P
 Show authors\.
-.
 .P
 \-\-no\-email
-.
 .P
 Don\'t show authors\' email\.
-.
 .SH "EXAMPLES"
-.
 .TP
 Updating AUTHORS file:
-.
-.IP
-$ git authors
-.
+
 .TP
 Listing authors:
-.
-.IP
-$ git authors \-\-list
-.
-.IP "" 4
-.
-.nf
-
-TJ Holowaychuk <tj@vision\-media\.ca>
```

Comment:
Why the authors listed here are removed?

Reply:
Emmm.., wait a moment, I miss this one cause it is an auto-task here.

Reply:
It looks the `*` under examples creates a warning `warn: unrecognized inline tag: ["p"]`, then something follows it was dropped. Funny...and remove it can recover it soon.

Reply:
Recovered

Reply:
The `<pre><code>` has been moved as `<p><code>`and warn at [inline_filter](https://github.com/apjanke/ronn-ng/blob/44db815d89d641b52e7fa5ae7df9951d5ed5b623/lib/ronn/roff.rb#L247), it should warn this when 'p' or '["p"]'. But I cannot figure out why a `p` was passed in...

I am not familiar with ruby, hope it useful.